metpy/calc/thermo.py

```diff
@@ -776,6 +776,44 @@ def psychrometric_vapor_pressure_wet(dry_bulb_temperature, wet_bulb_temperature,
             pressure * (dry_bulb_temperature - wet_bulb_temperature).to('kelvin'))
 
 
+@exporter.export
+@check_units('[dimensionless]', '[temperature]', '[pressure]')
+def mixing_ratio_from_relative_humidity(relative_humidity, temperature, pressure):
+    r"""Calculate the mixing ratio from relative humidity, temperature, and pressure.
+
+    Parameters
+    ----------
+    relative_humidity: `pint.Quantity`
+        Relative humidity in percent
+    temperature: `pint.Quantity`
+        Air temperature
+    pressure: `pint.Quantity`
+        Total atmospheric pressure
+
+    Returns
+    -------
+    `pint.Quantity`
+        Dimensionless mixing ratio
+
+    Notes
+    -----
+    Formula adapted from [Hobbs1977]_ pg. 74.
+
+    .. math:: w = \frac{RH}{100} w_s
+
+    * :math:`w` is mixing ratio
+    * :math:`RH` is relative humidity
+    * :math:`w_s` is the saturation mixing ratio
+
+    See Also
+    --------
+    relative_humidity_from_mixing_ratio, saturation_mixing_ratio
+
+    """
+    return (relative_humidity / (100 * units.percent) *
```

Comment:
For consistency with the rest of the library, we should take relative humidity as [0, 1]:

https://unidata.github.io/MetPy/latest/api/generated/metpy.calc.heat_index.html#metpy.calc.heat_index

You can pass in percent then using the units:
```python
mixing_ratio_from_relative_humidity(85 * units.percent, 25 * units.degC, 988 * units.hPa)
```

The docs for this function would need to updated to reflect this.

Reply:
That sounds good, thank you for the clarification. 

For background, I was simply basing this function off of how the existing `relative_humidity_wet_psychrometric`, `relative_humidity_from_mixing_ratio`, and `relative_humidity_from_specific_humidity` functions handle it, all of which use RH in percent. For consistency, I could update those as well if you would like.

Reply:
@jrleeman Yet again, we fail at code review.

@jthielen Thanks for pointing that out. It would be awesome if you would fix those too. Can you do those as a separate pull request? That will help ensure I don't forget to include those as an API change in the release notes for 0.7.